README.md

```diff
@@ -1,73 +1,72 @@
+| remote               | Defines information specific to the remote participant&#39;s public network interface. Only defined when the use case requires a single-host setup. | dict | no | - |
+| remote/ip            | A key to the remote dictionary defining the IP address of the remote participant&#39;s public network interface. | str | no | - |
+| remote/hostname      | A key to the remote dictionary defining the DNS hostname of the remote participant&#39;s public network interface | str | no | - |
+| remote/subnet        | A key to the remote dictionary defining the subnet of the remote participant&#39;s public network interface. | str | no | - |
+| remote/rsasigkey     | A key to the remote dictionary defining the public key for RSA signature authentication           | str         | no       | -                       |
+| remote/cert          | A key to the remote dictionary defining the certificate nickname of your certificate in the NSS database | str | no | - |
+| ike                  | Defines information specific to setting up the IKE protocol.                                      | dict        | no       | -                       |
+| ike/alg              | A key to the ike dictionary defining which encryption/authentication algorithm to use for the connection. | str | no | vpn\_enc\_alg                 |
+| ike/lifetime         | A key to the ike dictionary defining how long the keying channel of a connection should last before being renegotiated | str | no | vpn\_lifetime    |
+| ipsec                | Defines information specific to setting up IPSec protocol.                                        | dict        | no       | -                       |
+| ipsec/alg            | A key to the ipsec dictionary defining which encryption/authentication algorithm to use for the connection. | str | no | vpn\_enc\_alg |
+| ipsec/lifetime       | A key to the ipsec dictionary defining how long the keying channel of a connection should last before being renegotiated | str | no | vpn\_lifetime  |
+| mode                 | The type of the connection.                                                                       | str         | no       | With no NAT, default to &#39;transport&#39;. Otherwise, &#39;tunnel&#39; |
+| auth                 | How the two security gateways should authenticate each other                                      | str         | no       | -                       |
+| auto                 | What operation, if any, should be done automatically at startup                                   | str         | no       | -                       |
+| rekey                | Whether a connection should be renegotiated when it is about to expire                            | str         | no       | -                       |
+
+### Role Use Cases
+
+[Role requirements](https://issues.redhat.com/browse/RHELBU-538)
```

Comment:
external users will not be able to follow this link - you can omit it - the summary below is sufficient